Source/WebCore/css/BasicShapeConversion.h

```diff
@@ -49,8 +49,19 @@ class CSSToLengthConversionData;
 class CSSValue;
 class RenderStyle;
 
+struct Length;
+struct LengthPoint;
+struct LengthSize;
+
+enum class CoordinateAffinity : uint8_t;
```

Comment:
In other headers I’ve been sorting alphabetically so `enum class` goes between `class` and `struct`.